src/common/base/NebulaKeyUtils.h

```diff
@@ -103,12 +136,15 @@ class NebulaKeyUtils final {
     }
 
     static bool isDataKey(const folly::StringPiece& key) {
-        return !key.empty() && key[0] != kSysPrefix;
+        static int32_t typeMask = 0x000000FF;
+        static int32_t len = static_cast<int32_t>(sizeof(NebulaKeyType));
```

Comment:
Again, `constexpr` and no `static`.

Reply:
But I found `kVertexLen` and `kEdgeLen` are be modified by `static constexpr` ?